.github/workflows/cmx-complete.yaml

```diff
@@ -0,0 +1,117 @@
+name: Compatibilty Test with CMX in complete mode
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version of the chart to test'
+        required: true
+        default: '0.2.0'
+
+env:
+  chart_name: mlflow
+    
+jobs:
+  compatibility-matrix:
+    strategy:
+      fail-fast: false
+      matrix:
+        cluster: [ {distribution: kind, version: v1.25.3}, {distribution: k3s, version: v1.26} ]
+
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Package Helm Chart for Replicated
+        id: package-helm-chart
+        run: |
+          helm package applications/mlflow/charts/mlflow -u --version ${{ github.event.inputs.version }}
+
+      - name: Create Release
+        id: create-release
+        uses: replicatedhq/replicated-actions/create-release@v1
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          app-slug: ${{ secrets.REPLICATED_APP }}
+          chart: ${{env.chart_name}}-${{ github.event.inputs.version }}.tgz
+
+      - name: Create Customer
```

Comment:
Do we have a use case where you think it's necessary to create a unique customer for each distribution that we test against, or should we just have a single customer used for all of the pipelines? I'm not saying there's a right or no answer here. Just curious on your thoughts.